apps/commudle-admin/src/app/feature-modules/dashboard/components/user-dashboard/user-dashboard.component.html

```diff
@@ -0,0 +1,85 @@
+<div class="dashboard">
+  <div class="base-layout">
+    <div class="container">
+      <div class="two-column-layout">
+        <div class="left-column">
+          <commudle-user-profile [currentUser]="currentUser"></commudle-user-profile>
+          <nb-card class="user-profile-details">
+            <nb-card-body *ngIf="currentUser">
+              <commudle-userprofile-details
+                [userProfileDetails]="userProfileDetails"
+                [username]="currentUser.username"
+              ></commudle-userprofile-details>
+            </nb-card-body>
+          </nb-card>
+          <div class="your-communities">
```

Comment:
let's rename this class name to user-managed-communities, and similar to below classes name